galaxy.yml

```
@@ -2,13 +2,14 @@ namespace: ansible
 name: posix
 version: 0.1.0
 readme: README.md
-authors: null
+authors:
+  - Ansible (github.com/ansible)
```

Are you sure it shouldn't be an email field?
```suggestion
- Ansible <info@ansible.com>
```